root/layout/header/top.tt

```diff
@@ -2,16 +2,10 @@
     <ul class="menu">
     [%- IF c.user_exists -%]
         <li class="account">
-            [% link_entity(c.user) %]
+            <span class="menu-header">[%- c.user.name -%]</span>
```

Comment:
This does lose the gravatar (but we didn't have the gravatar before so it might not be a big deal?). Do we have any code currently that shows the gravatar but does not create a link to the user? If not, should we?